website/content/v1.0/guides/nvidia-gpu.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "NVIDIA GPU"
+description: "In this guide we'll follow the procedure to support NVIDIA GPU on Talos."
+---
+
+> Enabling NVIDIA GPU support on Talos is bound by [NVIDIA EULA](https://www.nvidia.com/en-us/drivers/nvidia-license/)
+> Talos GPU support is an **alpha** feature.
+
+There are three steps to enabling NVIDIA support in Talos.
+
+- Building a custom Talos installer image with NVIDIA modules
+- Building NVIDIA container toolkit system extension which allows to register a custom runtime with containerd
+- Upgrading Talos with the custom installer and enabling NVIDIA modules and the system extension
+
+Both these components require that the user build and maintain their own Talos installer image and the NVIDIA container toolkit [Talos System Extension](../system-extensions).
+
+## Prerequisites
+
+This guide assumes the user has access to a container registry with push permissions, docker installer on the build machine and the Talos host has access to the registry.
+
+Set the local registry and username environment variables:
+
+```bash
+export USERNAME=<username>
+export REGISTRY=<registry>
+export TAG=<custom installer image tag>
+```
```

Comment:
probably it makes sense to give some examples and take a note that one should be able to push to that registry (have credentials/auth)

Reply:
The auth part is mentioned here https://github.com/siderolabs/talos/pull/5230/files#diff-612c3f5d62e58a91892944466ccb1871e565255ee8b3c97e5487d3a742adfc17R19

Reply:
fixed